src/EventStore.Core.Tests/Common/ClusterNodeOptionsTests/when_building/with_cluster_node_and_custom_settings.cs

```diff
@@ -85,68 +85,15 @@ public class with_dns_discovery_disabled_and_gossip_seeds_defined<TLogFormat, TS
 
 	[TestFixture(typeof(LogFormat.V2), typeof(string))]
 	[TestFixture(typeof(LogFormat.V3), typeof(uint))]
-	public class with_prepare_ack_count_set_higher_than_the_quorum<TLogFormat, TStreamId> : ClusterMemberScenario<TLogFormat, TStreamId> {
-		protected override ClusterVNodeOptions WithOptions(ClusterVNodeOptions options) =>
-			options with {
-				Cluster = options.Cluster with {
-					PrepareCount = _quorumSize + 1
-				}
-			};
-
-		[Test]
-		public void should_set_prepare_count_to_the_given_value() {
-			Assert.AreEqual(_quorumSize + 1, _options.Cluster.PrepareAckCount);
-		}
-	}
 
 	[TestFixture(typeof(LogFormat.V2), typeof(string))]
 	[TestFixture(typeof(LogFormat.V3), typeof(uint))]
-	public class with_commit_ack_count_set_higher_than_the_quorum<TLogFormat, TStreamId> : ClusterMemberScenario<TLogFormat, TStreamId> {
-		protected override ClusterVNodeOptions WithOptions(ClusterVNodeOptions options) =>
-			options with {
-				Cluster = options.Cluster with {
-					CommitCount = _quorumSize + 1
-				}
-			};
-
-		[Test]
-		public void should_set_commit_count_to_the_given_value() {
-			Assert.AreEqual(_quorumSize + 1, _options.Cluster.CommitAckCount);
-		}
-	}
 
 	[TestFixture(typeof(LogFormat.V2), typeof(string))]
 	[TestFixture(typeof(LogFormat.V3), typeof(uint))]
-	public class with_prepare_ack_count_set_lower_than_the_quorum<TLogFormat, TStreamId> : ClusterMemberScenario<TLogFormat, TStreamId> {
-		protected override ClusterVNodeOptions WithOptions(ClusterVNodeOptions options) =>
-			options with {
-				Cluster = options.Cluster with {
-					PrepareCount = _quorumSize - 1
-				}
-			};
-
-		[Test]
-		public void should_set_prepare_count_to_the_quorum_size() {
-			Assert.AreEqual(_quorumSize, _options.Cluster.PrepareAckCount);
-		}
-	}
 
 	[TestFixture(typeof(LogFormat.V2), typeof(string))]
 	[TestFixture(typeof(LogFormat.V3), typeof(uint))]
```

Comment:
These `TestFixture` attributes should also be removed with the deleted tests

Reply:
TestFixture attributes removed.